apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -169,14 +170,20 @@ export const Timezones = {
 
 async function addHostsToDb(eventTypes: InputEventType[]) {
   for (const eventType of eventTypes) {
-    if (eventType.hosts && eventType.hosts.length > 0) {
-      await prismock.host.createMany({
-        data: eventType.hosts.map((host) => ({
-          userId: host.userId,
-          eventTypeId: eventType.id,
-          isFixed: host.isFixed ?? false,
-        })),
-      });
+    if (eventType.hosts) {
```

Comment:
Would this work to prevent more nesting?

```suggestion
    if (!eventType.hosts) continue;
```

Reply:
Also an empty array is still truthy so shouldn't we be checking for length as well?